### ISSUE TITLE
push_config.version must be one of [v1 v2]

### ISSUE THREAD

**Issue by leojkwan** [GetStream/stream-chat-js#949]
I noticed my chat feature is no longer working in production. I see this error in my jwt token firebase function logs:
`"push_config.version must be one of [v1 v2]"
    at StreamChat.errorFromResponse (/workspace/node_modules/stream-chat/dist/index.js:7299:15)`
    
    What am I missing? Using "stream-chat": "^6.4.0".

**Comment by FlameFractal** [comment]
I'm facing the same error on trying to follow this guide for migrating permissions from v1 to v2 ([Migrating From Legacy](https://getstream.io/chat/docs/react/migrating_from_legacy/?q=push_config))

```
await client.updateAppSettings({
  // @ts-ignore
  permission_version: 'v2',
  migrate_permissions_to_v2: true,
});
```

Error - `Unhandled Rejection Error: StreamChat error code 4: UpdateApp failed with error: "push_config.version must be one of [v1 v2]"`

If I do add `push_config.version` then it complains about firebase push configuration. But I don't want to add firebase, I just want to upgrade my permissions version

`Unhandled Rejection Error: StreamChat error code 4: UpdateApp failed with error: "firebase credentials are invalid"` 


Any clue?

**Comment by ferhatelmas** [comment]
Thanks! Closing but if you see any issues, let me know.

**Comment by leojkwan** [comment]
Still getting an error:
```
Unhandled error Error: StreamChat error code 4: UpdateApp failed with error: "cannot migrate permissions because app has been already switched to permissions v2"
```

**Comment by AnatolyRugalev** [comment]
@leojkwan this error means that your application was already switch to version 2 of permissions, so migration is not necessary and you don't have to send `migrate_permissions_to_v2` field in the request

**Comment by alexander01202** [comment]
Hello. I am getting the same error:
`Error: StreamChat error code 4:Unhandled Rejection Error: StreamChat error code 4: UpdateApp failed with error: "firebase credentials are invalid"`

I am not migrating from any version. Just trying to call:
```
const serverClient = StreamChat.getInstance( api_key, api_secret);
const apn_template = `{
    "aps": {
        "content-available": 1
    }
  }`;
  
  serverClient.updateAppSettings({
    firebase_config: {
      apn_template,
    }
  });
```

This all according to the [docs](https://getstream.io/chat/docs/sdk/reactnative/guides/push-notifications-v2/#make-ios-payload-data-only)

I have set the firebase credentials in the dashboard. I also implemented some code in my front end which need firebase intergration and they all work but on my backend (node js). It doesn't work. It keeps giving me this error.
